common/src/main/java/com/microsoft/identity/common/crypto/AndroidWrappedKeyLoader.java

```diff
@@ -99,9 +99,6 @@ public class AndroidWrappedKeyLoader extends AES256KeyLoader {
     private final CachedData<SecretKey> mKeyCache = new CachedData<SecretKey>() {
         @Override
         public SecretKey getData() {
-            if (!AndroidKeyStoreUtil.canLoadKey(mAlias) || !getKeyFile().exists()) {
```

Comment:
Should we add this check where we see encrypt/decrypt failures with wrapped key.
Thoughts? 
cc: @rpdome

Reply:
If we hit a decryption failure, then I would expect the key that's in memory (and was either loaded from persisted assets or generated fresh and then persisted this session) already doesn't match the one that got used on disk.  I think this would delete the persisted assets for the different, new key that isn't able to decrypt the old stuff, and then we might just wind up in the same cycle again.  What do you think?

Reply:
Closing as this one isn't relevant anymore.